### DISCUSSION TITLE
Welcome to centos2ol Discussions!

### DISCUSSION THREAD

**Discussion by Djelibeybi** [oracle/centos2ol#79] [Ideas]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dish67** [comment]
Hi 
When I launched the script, I encountered an error like this:
ERROR
--> Finished Dependency Resolution  
Error: Package: openscap-containers-1.2.17-15.0.3.el7_9.noarch (RONA_OL7_OL7-Latest)  
           Requires: python-docker  
 You could try using --skip-broken to work around the problem  
 You could try running: rpm -Va --nofiles --nodigest

I found the solution by removing the package. My question is, **how can I restart your script centos2ol.sh?** Because when I try, I get the following error:

#bash centos2ol.sh -k

Checking for required packages...  
Checking your distribution...  
You appear to be already running Oracle Linux.  
For assistance, please open an issue via GitHub: https://github.com/oracle/centos2ol/.
